modules/db2/src/main/java/org/testcontainers/containers/Db2Container.java

```diff
@@ -0,0 +1,94 @@
+package org.testcontainers.containers;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import org.testcontainers.DockerClientFactory;
+import org.testcontainers.containers.output.OutputFrame;
+import org.testcontainers.containers.output.WaitingConsumer;
+import org.testcontainers.utility.LogUtils;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+import java.util.function.Predicate;
+
+public class Db2Container<SELF extends Db2Container<SELF>> extends JdbcDatabaseContainer<SELF> {
+
+    static final String NAME = "db2";
+    static final String DEFAULT_DB2_IMAGE_NAME = "ibmcom/db2express-c";
+    static final String DEFAULT_TAG = "10.5.0.5-3.10.0";
+    private static final int DB2_PORT = 50000;
+
+    private String username = "db2inst1";
+    private String password = "foobar1234";
+
+    private static final int DEFAULT_STARTUP_TIMEOUT_SECONDS = 240;
+    private static final int DEFAULT_CONNECT_TIMEOUT_SECONDS = 240;
+
+    public Db2Container() {
+        this(DEFAULT_DB2_IMAGE_NAME + ":" + DEFAULT_TAG);
+    }
+
+    public Db2Container(String imageName) {
+        super(imageName);
+        withStartupTimeoutSeconds(DEFAULT_STARTUP_TIMEOUT_SECONDS);
+        withConnectTimeoutSeconds(DEFAULT_CONNECT_TIMEOUT_SECONDS);
+    }
+
+    @Override
+    protected void configure() {
+        addExposedPort(DB2_PORT);
+
+        addEnv("LICENSE", "accept"); // TODO: explicit?
+        addEnv("DB2INST1_PASSWORD", password);
+
+        withCommand("db2start");
```

Comment:
This should be set in the ctor, rather than here in `configure()`. Setting it here blocks any users from overriding the command (we had the same issue with the PostgreSQL module)